app/jobs/schedule_server_load_readings_job.rb

```diff
@@ -0,0 +1,22 @@
+class ScheduleServerLoadReadingsJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    Server.where(status: :up).each do |server|
+      cpu = SshExecution.new(server).
+            execute(command: "uptime | awk '{print $10}' | tr -d '\n'").to_f
+      cpu = 100 if cpu > 100
+      memory = SshExecution.new(server).
+               execute(command: "free | grep Mem | awk '{print $3/$2}' | tr -d '\n'").to_f
+      disk = SshExecution.new(server).
+             execute(command: "df | awk '{print $6,$5/100;}' | grep '^/ ' |  tr -d '\n /%'").to_f
+
+      ServerLoadReading.create(
```

Comment:
What do you think about using `create!` here. That way we get notified when there is something wrong.